### PR TITLE
Content update - Property changed events (user story 1878475)

### PR DESCRIPTION
**Pull request by v-trisshores**
@adegeo, this article PR is ready for your review.

[Internal preview link](https://review.docs.microsoft.com/en-us/dotnet/desktop/wpf/events/property-change-events?view=netdesktop-5.0&branch=pr-en-us-1350)

### CODE REVIEW

File: dotnet-desktop-guide/net/wpf/events/property-change-events.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+Property trigger syntax is analogous to an "if" statement in programming. If the trigger condition is true, then the "body" of the property trigger is "executed". The "body" of a property trigger isn't code, its markup. That markup is limited to using one or more <xref:System.Windows.Setter> elements to set other properties of the object where the style or template is applied.
```
Comment (adegeo) [inline dotnet-desktop-guide/net/wpf/events/property-change-events.md:67]: Terminology suggests that we use "run" over "execute" if we can.
```suggestion
Property trigger syntax is analogous to an "if" statement in programming. If the trigger condition is true, then the "body" of the property trigger is "run". The "body" of a property trigger isn't code, its markup. That markup is limited to using one or more <xref:System.Windows.Setter> elements to set other properties of the object where the style or template is applied.
```

**Review by adegeo — CHANGES_REQUESTED** [review]
Awesome. Few minor changes to implement, otherwise ready to go!

**Comment by v-trisshores** [comment]
@adegeo, thanks for your review! The suggested changes are committed.

**Review by adegeo — APPROVED** [review]

**Comment by adegeo** [comment]
Awesome, thanks!!

**Pull request merged**